src/Interpreters/GlobalSubqueriesVisitor.h

```diff
@@ -205,10 +205,19 @@ class GlobalSubqueriesMatcher
     }
 
 private:
+    static bool should_be_executed_globally(const Data & data)
```

Comment:
please stick to naming convention: [2. For the names of functions (methods), use camelCase beginning with a lowercase letter.](https://clickhouse.com/docs/en/development/style#names)